pkg/terraform/hcl/backend_test.go

```diff
@@ -1,78 +1,125 @@
 package hcl
 
 import (
+	"path"
 	"testing"
 
 	"github.com/snyk/driftctl/pkg/iac/config"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestHCL_getCurrentWorkspaceName(t *testing.T) {
+	cases := []struct {
+		name string
+		dir  string
+		want string
+	}{
+		{
+			name: "test with default workspace",
+			dir:  "testdata/default_workspace",
+			want: "default",
```

Comment:
That test case overlaps with `test with non-default workspace`. It test exactly the same code. Does it really make sense to maintain it ?

Reply:
It's not the same code since the workspace is different

Reply:
Yes I know, but does that really cover more code than the `test with non-default workspace` case ?
We can put anything in the content of the environment file, the covered code is always the same. 
I'm not sure to see what statements this case additionally covers.